lib/explorer/data_frame.ex

```diff
@@ -2865,8 +2865,8 @@ defmodule Explorer.DataFrame do
       naive_datetime = %NaiveDateTime{} ->
         LazySeries.new(:lazy, [naive_datetime], {:naive_datetime, :nanosecond})
 
-      datetime = %DateTime{} ->
-        LazySeries.new(:lazy, [datetime], {:datetime, :nanosecond})
+      datetime = %DateTime{time_zone: time_zone} ->
+        LazySeries.new(:lazy, [datetime], {:datetime, :nanosecond, time_zone})
```

Comment:
I think this was a bug, right?.

Also, shouldn't the precision be `:microsecond`?

Reply:
I think it should be microsecond, yeah. We should have a test for this. It should also be microsecond above if possible.

Reply:
[c278300](https://github.com/elixir-explorer/explorer/pull/924/commits/c2783003764c33055c3c00faa0f8ab85a753ff9c)